charts/refarch-templates/README.md

```
@@ -0,0 +1,295 @@
+              pathType: ImplementationSpecific
+      tls:
+        - secretName: my-tls-secret
+          hosts:
+            - my-example.local
+```
+
+#### Storage
+
+By default, no volumes are defined and no mounts are mounted connected to the containers. This can be useful to attach files like certificates.
+Example:
+```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)